app/uk/gov/hmrc/transitmovements/services/MessageFactory.scala

```diff
@@ -68,14 +74,29 @@ class MessageFactoryImpl @Inject() (
         Message(
           id = MessageId(ShortUUID.next(clock, random)),
           received = received,
-          generated = generationDate,
+          generated = Some(generationDate),
           messageType = messageType,
           triggerId = triggerId,
```

Comment:
when we create an empty message, we won't ever know the trigger id until later on, so can we remove it from the parameter list in the function and just set it to none in the object creation?

`def createEmptyMessage(
     messageType: MessageType,
     received: OffsetDateTime
   ): Message`

`triggerId = None,`